src/components/CreatePost/CreatePost.css

```diff
@@ -0,0 +1,137 @@
+.create-new-post
+{
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+}
+
+.whats-happening
+{
+  font-family: Comfortaa;
+  font-size: 16px;
```

Comment:
Suggestion: if you don't have different font size in parent container, you can remove this as 16px is default font size.

Reply:
Yeah, I need this to override some styling.